src/blocks/CTA/CTA.scss

```diff
@@ -19,56 +19,91 @@ $block: '.#{$namespace}cta';
 
     &__button {
         display: flex;
-        padding-bottom: $indentXS;
+        padding-bottom: 0px;
+
+        &_layout {
+            width: 100%;
+            max-width: 100%;
+            margin-right: 0px;
+
+            &_2 {
+                width: calc((100% / 2) - (#{$indentXS} / 2));
+                max-width: 50%;
+                margin-right: $indentXS;
+            }
+
+            &_2:nth-child(2n) {
+                margin-right: 0px;
+            }
+
+            &_3 {
+                width: calc((100% / 3) - (#{$indentM} / 3));
+                $max-width: 50%;
+                margin-right: $indentXS;
+            }
+
+            &_3:nth-child(3n) {
+                margin-right: 0px;
+            }
+
+            &_4 {
+                width: calc((100% / 4) - (#{$indentL} / 4));
+                max-width: 50%;
+                margin-right: $indentXS;
+            }
+
+            &_4:nth-child(4n) {
+                margin-right: 0px;
+            }
+        }
     }
 
     &__content {
         display: flex;
-        flex-direction: column;
+        flex-direction: row;
     }
-
-    @media (min-width: map-get($gridBreakpoints, 'sm')) {
+    @media (max-width: map-get($gridBreakpoints, 'md')) {
         &__content {
-            display: flex;
-            flex-direction: row;
+            flex-wrap: wrap;
         }
-
         &__button {
-            padding-bottom: 0px;
+            padding-bottom: $indentXS;
 
             &_layout {
-                width: 100%;
-                max-width: 100%;
-                margin-right: 0px;
-
-                &_2 {
+                &_1,
+                &_2,
+                &_3,
+                &_4 {
                     width: calc((100% / 2) - (#{$indentXS} / 2));
-                    max-width: 50%;
+                    max-width: 100%;
                     margin-right: $indentXS;
+                    flex-grow: 1;
                 }
 
-                &_2:nth-child(2n) {
-                    margin-right: 0px;
-                }
-
-                &_3 {
-                    width: calc((100% / 3) - (#{$indentM} / 3));
-                    $max-width: 50%;
-                    margin-right: $indentXS;
+                &_1:nth-child(2n),
+                &_2:nth-child(2n),
+                &_3:nth-child(2n),
+                &_4:nth-child(2n) {
+                    margin-right: 0;
                 }
+            }
+        }
+    }
+    @media (max-width: map-get($gridBreakpoints, 'sm')) {
+        &__content {
+            flex-direction: column;
+        }
 
-                &_3:nth-child(3n) {
-                    margin-right: 0px;
-                }
+        &__button {
+            padding-bottom: $indentXS;
 
+            &_layout {
+                &_1,
+                &_2,
+                &_3,
                 &_4 {
```

Comment:
can we set one common class for all four elements?

Reply:
+